libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -6864,6 +6864,7 @@ void GCS_MAVLINK::manual_override(RC_Channel *c, int16_t value_in, const uint16_
 void GCS_MAVLINK::handle_manual_control(const mavlink_message_t &msg)
 {
     if (msg.sysid != sysid_my_gcs()) {
+        GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "Ignore sysid %u", msg.sysid);
```

Comment:
1. this is only once place a warning might need to be sent (make a `send_sysid_ignored()` method?)
2. this message would need to be rate-limited as these messages can come in *fast*, and you will flood people with messages.